go.mod

```diff
@@ -310,3 +313,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
+
+replace github.com/sigstore/sigstore => github.com/hectorj2f/sigstore v1.1.1-0.20220328195805-4ade568cebda
```

Comment:
I am using my own fork until https://github.com/sigstore/sigstore/pull/353 gets approved.

Reply:
Changed to point to sigstore/sigstore main.